src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,17 +30,26 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'position_after_functions_and_oop_constructs' => self::NEXT_LINE,
```

Comment:
Regarding the boolean option, [other](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1fe4e61f89f48313ae229761c0da915acb2bea78/src/Fixer/Whitespace/NoSpacesAroundOffsetFixer.php#L39) [fixers](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/bf0f42981f8a89708ed9e16c33cb3a876d146f3d/src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php#L53) [seem](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/7651b56fab166dd9adf468886f07fdae97384ec7/src/Fixer/LanguageConstruct/DeclareEqualNormalizeFixer.php#L49) [to](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/f6101b0b733772fdf283c573591433a6828b3fc0/src/Fixer/Alias/NoMixedEchoPrintFixer.php#L52) [use](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/ec4b981a72075f10d2665c0ede7666f81df2b443/src/Fixer/Import/OrderedImportsFixer.php#L72) strings for binary choices as well. The booleans might just be confusing – the opposite of enforcing a brace placement is not-enforcing it, not enforcing other placement.

I address splitting the option in the OP, almost nobody wants to control the position of the class braces individually (according the [PSR-2 survey](www.php-fig.org/psr/psr-2/#a3-survey-results)) and it would only complicate the whole thing further.